Jenkinsfile

```diff
@@ -1,4 +1,11 @@
 openriscPipeline {
+
+    yosysReport {
+		core 'mor1kx'
+		target 'synth'
+		logPath 'build/mor1kx_5.0-r3/synth-icestorm/yosys.log'
```

Comment:
again here we can't keep the version in the path name.

Reply:
@stffrdhrn This should work now. @oleg-nenashev will be looking into https://ci.librecores.org/ stability maybe in a couple of weeks